.changeset/weak-roses-decide.md

```diff
@@ -0,0 +1,6 @@
+---
+'react-select': minor
+'@react-select/docs': patch
```

Comment:
How do we version docs? Should this be `minor` too?

Reply:
No one is consuming the docs as a package, so I probably wouldn't even worry about versioning it.

Reply:
I've noticed it's at version `3.1.2` though, and no idea how it got there 🤷